packages/fastboot/test/fastboot-test.js

```diff
@@ -50,41 +50,6 @@ describe('FastBoot', function() {
     expect(fn).to.throw(/(.+)package.json was malformed or did not contain a fastboot config/);
   });
 
-  it('can render HTML', function() {
```

Comment:
@xg-wang this test could be added to the `basic-app` test-package. 

I worry that is we have a lot of different test-packages then developers will not know where to add new tests.